packages/cashier/src/Containers/cashier.jsx

```diff
@@ -41,9 +41,11 @@ const Cashier = ({
     setTabIndex,
     tab_index,
     toggleCashier,
+    setIsCashierDefault,
 }) => {
     React.useEffect(() => {
         toggleCashier();
+        setIsCashierDefault(false);
```

Comment:
`setIsCashierDefault(false)` is already called when `CashierDefault` is unMount. So, I believe this is not the ideal solution.